src/components/remotecontrol/remotecontrol.js

```diff
@@ -598,6 +599,7 @@ export default function () {
 
     function onPlayPauseStateChanged() {
         updatePlayPauseState(this.paused(), true);
+        onPlaylistUpdate();
```

Comment:
IMO, updating the entire playlist is too much.
What if query `.playlistIndexIndicatorImage` and toggle the paused class?